plugin/health/health.go

```diff
@@ -29,8 +30,10 @@ func (h *health) OnStartup() error {
 		h.Addr = ":8080"
 	}
 	h.stop = make(chan bool)
-
-	ln, err := net.Listen("tcp", h.Addr)
+	// Reloading the plugin without changing the listening address results
+	// in an error unless we reuse the port because Startup is called for
+	// new handlers before Shutdown is called for the old ones.
```

Comment:
you don't have to repeat the comment everywhere

Reply:
Done